fdbclient/NativeAPI.actor.cpp

```diff
@@ -1743,6 +1743,34 @@ void setNetworkOption(FDBNetworkOptions::Option option, Optional<StringRef> valu
 	}
 }
 
+// update the network busyness on a 1s cadence
+ACTOR Future<Void> monitorNetworkBusyness() {
+	state double prevTime = now();
+	loop choose {
+		when(wait(delay(CLIENT_KNOBS->NETWORK_BUSYNESS_MONITOR_INTERVAL, TaskPriority::FlushTrace))) {
+			double elapsed = now() - prevTime; // get elapsed time from last execution
+			prevTime = now();
+			if (!g_network->isSimulated()) {
```

Comment:
I think this is probably ok to run in simulated clusters, though I could be wrong about that. You could see if simulation fails when you remove this check.